Remove eventids collection

c-w opened the pull request:
This is the schema change for the following service-side changes:
https://github.com/CatalystCode/project-fortis-services/pull/92
https://github.com/CatalystCode/project-fortis-services/pull/94

erikschlegel commented on ops/storage-ddls/cassandra-setup.cql:
```diff
@@ -175,27 +175,27 @@ CREATE TABLE popularplaces (
     pipelinekey text,
     externalsourceid text,
-    PRIMARY KEY ((placeid), conjunctiontopic1, conjunctiontopic2, conjunctiontopic3, pipelinekey, externalsourceid, centroidlat, centroidlon)
+    PRIMARY KEY ((placeid, conjunctiontopic1, conjunctiontopic2, conjunctiontopic3), pipelinekey, externalsourceid, eventtime, eventid)
```
What are you throughts on the following `PRIMARY KEY ((conjunctiontopic1, conjunctiontopic2, conjunctiontopic3, pipelinekey, externalsourceid), centroidlat, centroidlon, periodstartdate, periodenddate, eventid)` ?

erikschlegel replied:
We need to be able to filter off `centroidlat` and `centroidlon`. Please add them to the PK.

erikschlegel replied:
Remove `placeid` from the partition key as we're not filtering off it for all queries for this table. You can tack it on the PK.

c-w replied:
Not sure I understand the comments, could you elaborate?

We're querying this table in two places:

*Messages.byBbox*
<img src="https://user-images.githubusercontent.com/1086421/29181623-b53a6264-7db0-11e7-9e8f-e9403a79fc86.png" height=200 />

*Messages.byLocation*
<img src="https://user-images.githubusercontent.com/1086421/29181651-cdc99e8a-7db0-11e7-991e-4b926b772fff.png" height=200 />

In both cases we query on the placeid, not on the centroid locations.

c-w replied:
Also note that querying on placeid is more precise than centroid locations since there can be multiple places for the same location but every id is guaranteed to refer to only one location.

erikschlegel replied:
We should not be filtering off placeid. Please replace that filter with the centroid lat / long, which is based off the map bounds bbox.

erikschlegel replied:
The size of that placeid array would crush us:)

c-w replied:
For Messages.byLocation we're querying by a single place id. Wouldn't this be the most effective type of query that we have in Cassandra since it's a point query?

c-w replied:
@erikschlegel Updated the table definitions in f8aa433. See new query here https://github.com/CatalystCode/project-fortis-services/pull/94

c-w committed:
Remove eventids collection

This is the schema change for the following service-side changes:
https://github.com/CatalystCode/project-fortis-services/pull/92
https://github.com/CatalystCode/project-fortis-services/pull/94

erikschlegel reviewed (approved):
LGTM

c-w commented:
Looks like this was already merged in https://github.com/CatalystCode/project-fortis-pipeline/commit/1c2d47ed1307e6ec139c0521dfb5162f39206a2b
